daffodil-core/src/main/java/org/apache/daffodil/api/debugger/package-info.java

```diff
@@ -20,27 +20,21 @@
  *
  * <h2>Overview</h2>
  * <p>
- * Daffodil comes with one prebuilt debugger, the {@link org.apache.daffodil.api.debugger.TraceDebuggerRunner}, which outputs
+ * Daffodil comes with one prebuilt debugger, the TraceDebugger which is a 
+ * {@link org.apache.daffodil.api.debugger.Debugger}, which outputs
  * verbose information during the parsing processes, which can be used to aid
  * in debugging a DFDL schema. For example, the {@link
- * org.apache.daffodil.api.debugger.TraceDebuggerRunner} can be used like so:
+ * org.apache.daffodil.api.debugger.Debugger} TraceDebugger can be used like so:
  *
  * <pre>
  * {@code
- * TraceDebuggerRunner tdr = InteractiveDebuggerRunnerFactory.getTraceDebuggerRunner(System.out);
- * Daffodil.setDebugger(tdr);
+ * Debugger td = Daffodil.newTraceDebugger(System.out);
+ * dp.setDebugger(td);
  * }</pre>
  * <p>
- * Additionally, one may create their own debugger runner by implementing the
+ * Additionally, one may create their own debugger by implementing the
```

Comment:
I think we need to make it clear that there are kindof two different ways to create a debugger, where this only really discusses one.

One way is to create a class that implements the Debugger interface.

The other way is to create a class that implements the InteractiveDebuggerRunner interface, and call Daffodil.withInteractiveDebugger(...) to get a Debugger.

Regardless of which way is chose, the Debugger can then be used by calling Daffodil.withDebugger(...)